.travis.yml

```diff
@@ -12,33 +12,32 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
  
 env:
-  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+  global:
+    - SYMFONY_DEPRECATIONS_HELPER=309
 
 matrix:
   include:
     - php: 5.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=2.3.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=309
+      env: SYMFONY_VERSION=2.7.*
     - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
```

Comment:
the 3.0 build was removed as master does not support 3.0 and we have no plan to support it. Still executing the job only means reducing test speed of the complete CMF org (as an org is only allowed to have 6 concurrent jobs)